### ISSUE TITLE
F-4 missing fuselage parts

### ISSUE THREAD

**Issue by Kerbal410** [Drag0nD3str0yer/Moderately-Plane-Related#23]
the F-4 cockpit doesn't fit any existing parts in KSP. Mk0 is a bit small, and Mk1 is a bit big. the same goes for the bottom section of the cockpit, as it is flat. please help.

**Comment by Drag0nD3str0yer** [comment]
Yes, Mk0.5 is in the plans to be added, for the immediate term, just rely on Tweakscale. and for the F-4 cockpit, it is designed to be used with Procedural Wings, but if you need to, just use normal fuselage parts to sort of hide the bottom section.

And yes, i do accept part suggestions, but the backlog is fairly long, so it may be a while.

**Comment by Kerbal410** [comment]
thanks. good luck with the IRST
